app/js/components/searchHistory/searchHistory.css

```diff
@@ -15,28 +15,22 @@
     background-color: #ccc;
     color: #000;
     padding: 10px;
-    border-color: #000;
-    border-width: 1px;
-    border-style: solid;
     margin-top: 20px;
     margin-bottom: 20px;
 }
 
 .result-window {
     padding: 10px;
-    border-color: #000;
-    border-width: 1px;
-    border-style: solid;
```

Comment:
Well I would have preferred to use a single property but I didn't write this part and we had to even take it off